docs/src/atoms/dropdown.js

```diff
@@ -1,18 +1,18 @@
 'use strict';
 
-import 'images/avatar1.jpg';
-import 'images/avatar2.jpg';
-import 'images/avatar3.jpg';
-import 'images/avatar4.jpg';
-import 'images/avatar5.jpg';
+import 'global/images/avatar1.jpg';
+import 'global/images/avatar2.jpg';
+import 'global/images/avatar3.jpg';
+import 'global/images/avatar4.jpg';
+import 'global/images/avatar5.jpg';
 
 import { Card, Dropdown, Grid, Header, SubNavigation, TitleBar } from 'react-cm-ui';
-import Block from '../app/block.js';
-import Highlighter from '../app/highlighter.js';
-import Main from '../app/main.js';
+import Block from '../global/block.js';
+import Highlighter from '../global/highlighter';
+import Main from '../global/main';
 import PropTypes from 'prop-types';
 import React from 'react';
-import TableProps from '../app/tableProps';
+import TableProps from '../global/tableProps';
 
 const getImageUrl = (imageFileName) => window.location.host.indexOf('localhost') > -1 ?
     `/_/docs/src/images/${imageFileName}` :
```

Comment:
We gotta adjust this too, if we move the images, right?

Reply:
Fixed.